server/test/unit/paTest/getGooglePayloadTest.js

```diff
@@ -0,0 +1,100 @@
+// Third-party dependencies
+const { expect, use } = require('chai')
+const { describe, it } = require('mocha')
+const sinon = require('sinon')
+const sinonChai = require('sinon-chai')
+
+// In-house dependencies
+const { helpers, googleHelpers } = require('brave-alert-lib')
+const { mockResponse } = require('../../../testingHelpers')
+const pa = require('../../../pa')
+
+use(sinonChai)
+
+const sandbox = sinon.createSandbox()
+
+const fakePaPayload = {
+  aud: 'fakeclientid.apps.googleusercontent.com',
+  iss: 'https://accounts.google.com',
+  exp: Date.now(),
+  hd: 'brave.coop',
+  email: 'brave@brave.coop',
+  name: 'Brave User',
+}
+
+describe('pa.js unit tests: getGooglePayload', () => {
+  beforeEach(() => {
+    sandbox.spy(helpers, 'log')
+    sandbox.spy(pa, 'getGooglePayload')
```

Comment:
I don't see where you use the spy capabilities for `pa.getGooglePayload`. Am I missing something?

Reply:
gotcha! Removed!